README.md

```diff
@@ -173,6 +173,51 @@ Your email will be acknowledged within 24 hours, and you’ll receive a more
 detailed response to your email within 48 hours indicating the next steps in
 handling your report.
 
+There is not any hard and fast rule to determine if a bug is worth reporting as
+a security issue. The general rule is any issue worth reporting via the
+security address must allow an attacker to affect the confidentiality, integrity
+or availability of the Node.js application or its system for which the attacker
+does not already have the capability.
```

Comment:
If you have the right to run arbitrary js code, calling process.exit() isn't a vuln. If you have the mysql credentials, being able to dump the DB isn't a vuln. If you need shell access to execute an exploit that gives you shell access, its  not a vuln. I think this is important, a vulnerability allows capability escalation, if you already have the capability, it isn't a vuln.